Documentation/tutorial/links_and_cross_references.md

```diff
@@ -0,0 +1,269 @@
+\- file1.md
+```
+
+You can use relative path to reference `file2.md` in `file1.md`:
+
+```markdown
+[file2](subfolder/file2.md)
+```
+
+DocFX converts it to a relative path in output folder structure:
+
+```html
+<a href="subfolder/file2.html">file2</a>
+```
+
+You can see the source file name (`.md`) is replaced with output file name (`.html`).
+
+> DocFX does not simply replace the file extension here (`.md` to `.html`), it also tracks the mapping between input and output files to make sure source file path will resolve to correct output path. For example, if in the above case, `subfolder` is renamed to `subfolder2` using [file mapping](docfx.exe_user_manual.md#4-supported-name-files-file-mapping-format) in `docfx.json`, in output html, the link url will also resolve to `subfolder2/file2.html`.
+
+### Relative path vs. absolute path
+
+It's recommended to always use relative path to reference another file in the same project. Relative path will be resolved during build and produce build warning if the target file does not exist.
+
+> Tip: File must be included in `docfx.json` be processed by DocFX, so if you see a build warning about broken link but the file actually exists in your file system, go and check whether this file is included in `docfx.json`.
+
+You can also use absolute path (path starts with `/`) to link to another file, but DocFX won't check its correctness for you and will keep it as-is in the output HTML.
+That means you should use the output file path as absolute path. For example, in the above case, you can also write the link as follows:
+
+```markdown
+[file2](/subfolder/file2.html)
+```
+
+Sometimes you may find it's complicated to calculate relative path between two files.
+DocFX also supports path starts with `~` to represent path relative to the root directory of your project (i.e. where `docfx.json` is located).
+This kind of path will also be validated and resolved during build. For example, in the above case, you can write the following links in `file2.md`:
+ 
+```markdown
+[file1](~/file1.md)
+
+[file1](../file1.md)
+```
+
+Both will resolve to `../file1.html` in output html.
+
+> [Automatic link](https://daringfireball.net/projects/markdown/syntax#autolink) doesn't support relative path. If you write something like `<file.md>`, it will be treated as an HTML tag rather than a link.
+
+### Links in file includes
+
+If you use [file include](../spec/docfx_flavored_markdown.md#file-inclusion) to include another file, the links in included file is relative to the included file. For example, if `file1.md` includes `file2.md`:
+
+```markdown
+[!include[file2](subfolder/file2.md)]
+```
+
+All links in `file2.md` are relative to the `file2.md` itself, even when it's included by `file1.md`.
+
+Please note the file path in include syntax are handled differently than Markdown link. You can only use relative path to specify location of the included file. And it's not required included file to be included in `docfx.json`.
```

Comment:
`it's not required included file to be included` => `it's not required for included files to be included`